app/models/presenters/presenter.rb

```diff
@@ -82,6 +82,10 @@ def inspect
     "<#{self.class.name} labware:#{labware.uuid} ...>"
   end
 
+  def child_assets
+    nil
+  end
```

Comment:
What was the reason of using ```child_assets``` and the previous alias with ```child_plates```? Did child_plates didn't get the right value for the condition at ```app/models/concerns/presenters/statemachine/state_allows_single_child_creation.rb```?

Reply:
I included `child_assets` on the `Presenter` concern rather than the `PlatePresenter` class in order that the single child creation workflow can be applied to other types of presenters in the future if needed

Reply:
Yes, good idea 👍